lib/LaTeXML/Common/Error.pm

```diff
@@ -285,6 +285,8 @@ sub Fatal {
 
 sub hardYankProcessing {
   my $state = $STATE;
+  # Nothing we can do if we are called without a global $STATE bound
+  return unless $state;
```

Comment:
benefits of moving this code out into a subroutine - I can now stop early when the state is missing, e.g. if Fatal is called from some obscure place in initialization, post-processing, makesite ...